src/common/utils/model.ts

```diff
@@ -23,10 +25,14 @@ export function fetchProjections(
   }
 
   async function fetch() {
-    const summaryWithTimeseriesMap = await new Api(
+    const summaryWithTimeseries = await new Api(
       snapshotUrl,
-    ).fetchSummaryWithTimeseriesMap(region);
-    return new Projections(summaryWithTimeseriesMap, stateId, countyInfo);
+    ).fetchSummaryWithTimeseries(region);
+    assert(
+      summaryWithTimeseries != null,
+      'Failed to fetch projections for ' + region,
+    );
```

Comment:
Note: This assert is kinda' new.  We used to allow for missing API projections because we only produced them for locations where we had future projections, but when we added case incidence, the backend started producing API projections for all locations (even if we don't have any data / metrics).  So now we consider it an invariant that the API should have data for every state / county and we assert otherwise.